tokio/src/process/mod.rs

```diff
@@ -573,6 +573,19 @@ impl Command {
         self
     }
 
+    /// Set executable argument
+    ///
+    /// Set the first process argument, `argv[0]`, to something other than the
+    /// default executable path.
+    #[cfg(unix)]
```

Comment:
This attribute adds an annotation in the documentation that says that the function is only available on unix.
```suggestion
    #[cfg(unix)]
    #[cfg_attr(docsrs, doc(cfg(unix)))]
```